apps/site/assets/ts/stop/components/RouteCard.tsx

```diff
@@ -35,6 +35,7 @@ const RouteCard = ({
       hasAlert={
         alerts && alerts.filter(isHighSeverityOrHighPriority).length > 0
       }
+      numberOfAlerts={alerts.filter(isHighSeverityOrHighPriority).length}
```

Comment:
suggestion(non-blocking): The logic is sound. But it feels simpler to pass in `alerts={alerts.filter(isHighSeverityOrHighPriority)}` and then compute the presence of high-priority alerts and the number of them inside the `<RouteCardHeader />` component.